test/integration/http_integration.cc

```diff
@@ -360,8 +360,9 @@ void HttpIntegrationTest::initialize() {
   registerTestServerPorts({"http"}, test_server_);
 
   // Needs to outlive all QUIC connections.
+  std::shared_ptr<Upstream::MockClusterInfo> cluster{new NiceMock<Upstream::MockClusterInfo>()};
   auto quic_connection_persistent_info =
-      std::make_unique<Quic::PersistentQuicInfoImpl>(*dispatcher_, 0);
+      Quic::createPersistentQuicInfoForCluster(*dispatcher_, *cluster);
```

Comment:
Out of curiosity, why was this change needed? I'm guessing your PR exposed that the old code was doing something wrong?

Reply:
Yeah so normally a PersistentQuicInfo is created via this createPersistentQuicInfoForCluster() call https://github.com/envoyproxy/envoy/blob/dd141e13da7acb5b9788a46d366e3bcf6b38bd3a/source/common/http/http3/conn_pool.cc#L82, where a couple of configs are added on top of the vanilla PersistentQuicInfo.
My change of adding RVCM is one of them.